main/config.c

```diff
@@ -2926,10 +3006,12 @@ int ast_config_text_file_save2(const char *configfile, const struct ast_config *
 				for (incl=cfg->includes; incl; incl = incl->next) {
 					if (strcmp(incl->include_location_file, var->file) == 0){
 						if (var->lineno > incl->include_location_lineno && !incl->output) {
-							if (incl->exec)
-								fprintf(f,"#exec \"%s\"\n", incl->exec_file);
-							else
+							if (incl->include_type == 0)
 								fprintf(f,"#include \"%s\"\n", incl->included_file);
+							else if (incl->include_type < 0)
```

Comment:
Since this is new code / modifications, braces around the if/else

Reply:
Reworked